README.md

```diff
@@ -346,8 +346,13 @@ end
 
 You may specify that a plugin requires one or more Luarocks packages using the `rocks` key. This key
 takes either a string specifying the name of a package (e.g. `rocks=lpeg`), or a list specifying one or more packages.
-Entries in the list may either be strings or lists --- the latter case is used to specify a
-particular version of a package, e.g. `rocks = {'lpeg', {'lua-cjson', '2.1.0'}}`.
+Entries in the list may either be strings, a list of strings or a table --- the latter case is used to specify arguments such as the
```

Comment:
I haven't yet looked to see if this PR allows all valid `luarocks` arguments to be specified, but, if it does not, then we ought to state which arguments are allowed here.

Reply:
Will do 👍🏿 , I only included the arguments that made sense to me to be able to set in packer, some seemed like they could mess things up e.g. `--tree` or `--local`. The full list is [here](https://github.com/luarocks/luarocks/wiki/luarocks).

Reply:
Yup, supporting a limited set seems wise to me - I just want to make sure we document it. Thanks!